src/main/java/org/sagebionetworks/template/CloudFormationClientImpl.java

```diff
@@ -214,6 +214,7 @@ public Stack waitForStackToComplete(String stackName) throws InterruptedExceptio
 			case CREATE_IN_PROGRESS:
 			case UPDATE_IN_PROGRESS:
 			case UPDATE_COMPLETE_CLEANUP_IN_PROGRESS:
+			case UPDATE_ROLLBACK_COMPLETE:
```

Comment:
This will not work.  With this change it will just get stuck waiting for the status to change but it never will.  It needs to be treated as either a failure or success.  I believe treating it as a failure is the correct thing to do here.

Reply:
Is it not transient? From SDK doc on continueRollback(): "For a specified stack that is in the UPDATE_ROLLBACK_FAILED state, continues rolling it back to the UPDATE_ROLLBACK_COMPLETE state. Depending on the cause of the failure, you can manually fix the error and continue the rollback. By continuing the rollback, you can return your stack to a working state (the UPDATE_ROLLBACK_COMPLETE state), and then try to update the stack again.". My thinking is that the first time(s) through the loop it is still in UPDATE_ROLLBACK_COMPLETE, then at some point it would go the UPDATE_COMPLETE.

Reply:
See also https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-describing-stacks.html.